sample_client/src/main/java/com/owncloud/android/lib/sampleclient/ProgressBarAnimation.java

```diff
@@ -0,0 +1,40 @@
+package com.owncloud.android.lib.sampleclient;
+
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.content.Intent;
+import android.view.animation.Animation;
+import android.view.animation.Transformation;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+public class ProgressBarAnimation extends Animation {
+
+    private Context context;
+    private ProgressBar progressBar;
+    private TextView textView;
+    private Float from;
+    private Float to;
```

Comment:
```suggestion
    private Context mContext;
    private ProgressBar mProgressBar;
    private TextView mTextView;
    private Float mFrom;
    private Float mTo;
```
it would be great if you follow code style(see others files as well)